src/plugins/vis_builder/public/application/components/right_nav.tsx

```diff
@@ -28,6 +33,10 @@ export const RightNav = () => {
   const dispatch = useTypedDispatch();
   const StyleSection = ui.containerConfig.style.render;
 
+  const { activeVisualization } = useTypedSelector((state) => state.visualization);
+  const oldAggParams = activeVisualization?.aggConfigParams ?? [];
```

Comment:
Would this be safer with `||` rather than `??`?

Reply:
in this particular instance i dont think it really matters since the falsy value that we are defaulting here is always `undefined`. But in general i prefer nullish coalescing over logical or unless we know what other falsy values we want to default.

Reply:
Make sense. Changed `oldAggParams` to `aggConfigParams`